lms/static/sass/views/_teams.scss

```diff
@@ -148,6 +149,200 @@
     @extend %t-weight4;
   }
 
+  .search-tools {
+    @extend %t-copy-sub1;
+    margin: ($baseline/10) $baseline;
+    color: $gray-l1;
+
+    p, span { // override
+      color: inherit;
+    }
+  }
+
+  .card {
+    @extend .clearfix;
```

Comment:
Hmm, to not pollute our compiled CSS, can we use a `` @extend %clearfix;`` placeholder or just the old-fashioned Bourbon (see what I did there?) ``include clearfix()`` mixin utility?